torch/fx/experimental/proxy_tensor.py

```diff
@@ -433,13 +436,30 @@ def create_arg(self, a: Any):
         return super().create_arg(a)
 
 
+DISPATCH_TRACE_ERROR_PRINT_GRAPH = False
+
 def dispatch_trace(
         root: Union[torch.nn.Module, Callable],
         tracer: Tracer,
         concrete_args: Optional[Tuple[Any, ...]] = None,
 ) -> GraphModule:
-    graph = tracer.trace(root, concrete_args)
     name = root.__class__.__name__ if isinstance(root, torch.nn.Module) else root.__name__
+    try:
+        graph = tracer.trace(root, concrete_args)
+    except Exception:
+        if DISPATCH_TRACE_ERROR_PRINT_GRAPH:
+            gm_printed = GraphModule(tracer.root, tracer.graph, name).print_readable(print_output=False)
+            log.info(f"""
```

Comment:
info severity is too low, by default our log level is warning not info, so this won't actually show up under the instructions you gave below. If you want to keep the log level this way, then you should probably gate the logging on log.isEnabledFor(INFO), unconditionally print it, and then ask the user to crank up log level. Please... test if the API is working?

But I prefer a specialized enable trace flag, because if I turn on INFO I'm likely to get a bunch of other spew I don't really care about, but if I'm following up on the breadcrumb I saw below, I probably only care about this output. I suppose we can compromise on the earlier code review comment: the breadcrumb should be a log, while the actual triggered print should just be a print.

Reply:
I will not give you any trouble if you follow the functorch convention (although I think functorch is bad for spewing the graphs to stdout, fwiw.)